src/components/DoctorsPage/DoctorsCard/Schedule/Schedule.tsx

```diff
@@ -79,15 +63,30 @@ const Schedule: React.FC<IScheduleProps> = ({ workTimeData }) => {
       <WrapperHeader>
         <span>Doctors schedule</span>
       </WrapperHeader>
-      {workTimeData.length == 0 && <span>The doctor hasn't added a schedule yet.</span>}
+
+      {!workTimeData?.length && <span>The doctor hasn't added a schedule yet.</span>}
+
       <ItemWrapper>{renderItem()}</ItemWrapper>
+
       {stateSchedule.viewSuccessMessage && (
         <SuccessMessage>
-          You have booked a ticket for {stateSchedule.choiceWorkTime.time} on{' '}
-          {stateSchedule.choiceWorkTime.date}. Be healthy!
+          {error ? (
+            <Error errorMessage={error} />
```

Comment:
why error is contained in `SuccessMessage`? That's confusing

Reply:
renamed commponent to SuccessOrErrorMessage

Reply:
@KLIM-nk01 are these components different in styles?